### DISCUSSION TITLE
asdf-pkl

### DISCUSSION THREAD

**Discussion by chrisvander** [mise-plugins/.github#3] [Transfer Request]
Hi all, submitting a plugin I wrote for Apple's recently released Pkl configuration language. It's hosted under [chrisvander/asdf-pkl](https://github.com/chrisvander/asdf-pkl).

**Comment by jdx** [comment]
great! go ahead and start the transfer and I'll accept it

**Comment by chrisvander** [comment]
Getting `You don’t have the permission to create public repositories on mise-plugins` when I attempt to transfer, should I just transfer it to you?

**Reply by jdx** [reply]
oh hmm, yeah just transfer it to me and I'll put it in here
